web-frontend/src/pages/Welcome/Welcome.tsx

```diff
@@ -44,7 +44,7 @@ export default function Welcome() {
             <Button
               variant="secondary"
               onClick={() => {
-                handleImport.mutate({} as AccountObject);
+                mutate({} as AccountObject);
```

Comment:
Non blocking: 

Are we mutating something empty here!? 🤔 An empty object!? Is that the case!?

Don't we need any kind of file validation in FE at this point? If yes,  I would suggest use a **handleSubmit|Import** function and/or wrap that in a form. This allow us to proceed some validations if needed. 

However if we have not here, ok for me.

Reply:
it's just a PUT endpoint to wake up wails prompt.